web-ui/src/components/checkin/CheckinHistory.jsx

```diff
@@ -91,7 +94,7 @@ const CheckinsHistory = ({ history }) => {
   const DateInput = React.forwardRef((props, ref) => (
     <div className="date-input" ref={ref}>
       <p style={{ margin: "0px" }}>{props.value}</p>
-      <CalendarTodayIcon onClick={props.onClick}>stuff</CalendarTodayIcon>
+      <CalendarTodayIcon onClick={props.onClick}></CalendarTodayIcon>
```

Comment:
Minor, but this could be self-closing `<CalendarTodayIcon onClick={props.onClick} />`